### ISSUE TITLE
Invalid memory address or nil pointer dereference [compute]

### ISSUE THREAD

**Issue by guidorugo** [mlabouardy/komiser#108]
``` 
$ wget https://cli.komiser.io/2.3.0/linux/komiser
$ sudo chmod +x komiser
$ ./komiser start --port 3000 --redis redis_node_in_aws:6379 --duration 10 
2019/07/16 16:50:44 Successfully connected to Redis

10.200.150.121 - - [16/Jul/2019:16:50:51 +0100] "GET / HTTP/1.1" 200 2856
10.200.150.121 - - [16/Jul/2019:16:50:51 +0100] "GET /inline.31e1fb380eb7cf3d75b1.bundle.js HTTP/1.1" 200 798
10.200.150.121 - - [16/Jul/2019:16:50:51 +0100] "GET /styles.97e9f56065354e4bba5b.bundle.css HTTP/1.1" 200 643
10.200.150.121 - - [16/Jul/2019:16:50:51 +0100] "GET /polyfills.8e93ac3f450806a119ad.bundle.js HTTP/1.1" 200 63719
10.200.150.121 - - [16/Jul/2019:16:50:51 +0100] "GET /main.db6e3a4a520332bcb602.bundle.js HTTP/1.1" 200 350965
10.200.150.121 - - [16/Jul/2019:16:50:51 +0100] "GET /scripts.da8efe539c15fe8a4dea.bundle.js HTTP/1.1" 200 645368
10.200.150.121 - - [16/Jul/2019:16:50:51 +0100] "GET /vendor.e3e9ef62a125c66f7cd6.bundle.js HTTP/1.1" 200 1789044
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/profiles HTTP/1.1" 200 2
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/ec2/regions HTTP/1.1" 200 534
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/iam/users HTTP/1.1" 200 2
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/iam/account HTTP/1.1" 200 206
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/support/open HTTP/1.1" 500 44
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/support/history HTTP/1.1" 500 44
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/cost/current HTTP/1.1" 200 18
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/cost/history HTTP/1.1" 200 8491
10.200.150.121 - - [16/Jul/2019:16:50:56 +0100] "GET /favicon.ico HTTP/1.1" 200 1150
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/cost/forecast HTTP/1.1" 200 20
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/resources/regions HTTP/1.1" 200 37
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/cloudwatch/alarms HTTP/1.1" 200 20
10.200.150.121 - - [16/Jul/2019:16:50:55 +0100] "GET /aws/cloudwatch/alarms HTTP/1.1" 200 20
10.200.150.121 - - [16/Jul/2019:16:51:10 +0100] "GET /aws/ec2/regions HTTP/1.1" 200 534
2019/07/16 16:51:11 http: panic serving 10.200.150.121:58589: runtime error: invalid memory address or nil pointer dereference
goroutine 11 [running]:
net/http.(*conn).serve.func1(0xc420459ae0)
	/usr/local/go/src/net/http/server.go:1726 +0xd0
panic(0x1ea5ee0, 0x448e770)
	/usr/local/go/src/runtime/panic.go:502 +0x229
github.com/mlabouardy/komiser/services/aws.AWS.getClusters(0xc4203c3780, 0xa, 0x25bbbc0, 0xc420d95740, 0x25b8a60, 0xc4202cbd60, 0xc4204f7ce0, 0xc420dfb800, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:50 +0xfd
github.com/mlabouardy/komiser/services/aws.AWS.DescribeECS(0xc4202cd730, 0x9, 0x25bbbc0, 0xc420d95740, 0x25b8a60, 0xc4202cbd60, 0xc4204f7ce0, 0xc420dfb800, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:20 +0x3b0
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler(0xc42036c520, 0x7f97d931f188, 0xc4204f7980, 0xc420d80d00)
	/go/src/github.com/mlabouardy/komiser/handlers/aws/ecs_handler.go:23 +0x2cb
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler-fm(0x7f97d931f188, 0xc4204f7980, 0xc420d80d00)
	/go/src/github.com/mlabouardy/komiser/main.go:78 +0x48
net/http.HandlerFunc.ServeHTTP(0xc420337730, 0x7f97d931f188, 0xc4204f7980, 0xc420d80d00)
	/usr/local/go/src/net/http/server.go:1947 +0x44
github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc42033b730, 0x7f97d931f188, 0xc4204f7980, 0xc420d80d00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux/mux.go:162 +0xed
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.(*cors).ServeHTTP(0xc4204279e0, 0x7f97d931f188, 0xc4204f7980, 0xc42038eb00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/cors.go:51 +0xa2e
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.loggingHandler.ServeHTTP(0x25bb580, 0xc42000e020, 0x25ba920, 0xc4204279e0, 0x25c4a20, 0xc42075c380, 0xc42038eb00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/handlers.go:69 +0x123
net/http.serverHandler.ServeHTTP(0xc42020ea90, 0x25c4a20, 0xc42075c380, 0xc42038eb00)
	/usr/local/go/src/net/http/server.go:2694 +0xbc
net/http.(*conn).serve(0xc420459ae0, 0x25c5420, 0xc42032e900)
	/usr/local/go/src/net/http/server.go:1830 +0x651
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2795 +0x27b
10.200.150.121 - - [16/Jul/2019:16:51:10 +0100] "GET /aws/lambda/errors HTTP/1.1" 200 575
10.200.150.121 - - [16/Jul/2019:16:51:10 +0100] "GET /aws/lambda/invocations HTTP/1.1" 200 608
10.200.150.121 - - [16/Jul/2019:16:51:10 +0100] "GET /aws/lambda/functions HTTP/1.1" 200 22
10.200.150.121 - - [16/Jul/2019:16:51:11 +0100] "GET /aws/ec2/scheduled HTTP/1.1" 200 1
10.200.150.121 - - [16/Jul/2019:16:51:18 +0100] "GET /aws/cost/instance_type HTTP/1.1" 200 3792
2019/07/16 16:51:19 http: panic serving 10.200.150.121:58606: runtime error: invalid memory address or nil pointer dereference
goroutine 460 [running]:
net/http.(*conn).serve.func1(0xc42039f860)
	/usr/local/go/src/net/http/server.go:1726 +0xd0
panic(0x1ea5ee0, 0x448e770)
	/usr/local/go/src/runtime/panic.go:502 +0x229
github.com/mlabouardy/komiser/services/aws.AWS.getClusters(0xc420e04020, 0xa, 0x25bbbc0, 0xc420fdb9e0, 0x25b8a60, 0xc420fd9000, 0xc420ff4390, 0xc420f3f900, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:50 +0xfd
github.com/mlabouardy/komiser/services/aws.AWS.DescribeECS(0xc420fcd230, 0x9, 0x25bbbc0, 0xc420fdb9e0, 0x25b8a60, 0xc420fd9000, 0xc420ff4390, 0xc420f3f900, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:20 +0x3b0
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler(0xc42036c520, 0x7f97d931f188, 0xc420ff41b0, 0xc420fc0600)
	/go/src/github.com/mlabouardy/komiser/handlers/aws/ecs_handler.go:23 +0x2cb
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler-fm(0x7f97d931f188, 0xc420ff41b0, 0xc420fc0600)
	/go/src/github.com/mlabouardy/komiser/main.go:78 +0x48
net/http.HandlerFunc.ServeHTTP(0xc420337730, 0x7f97d931f188, 0xc420ff41b0, 0xc420fc0600)
	/usr/local/go/src/net/http/server.go:1947 +0x44
github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc42033b730, 0x7f97d931f188, 0xc420ff41b0, 0xc420fc0600)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux/mux.go:162 +0xed
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.(*cors).ServeHTTP(0xc4204279e0, 0x7f97d931f188, 0xc420ff41b0, 0xc420d13800)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/cors.go:51 +0xa2e
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.loggingHandler.ServeHTTP(0x25bb580, 0xc42000e020, 0x25ba920, 0xc4204279e0, 0x25c4a20, 0xc420e037a0, 0xc420d13800)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/handlers.go:69 +0x123
net/http.serverHandler.ServeHTTP(0xc42020ea90, 0x25c4a20, 0xc420e037a0, 0xc420d13800)
	/usr/local/go/src/net/http/server.go:2694 +0xbc
net/http.(*conn).serve(0xc42039f860, 0x25c5420, 0xc42080db80)
	/usr/local/go/src/net/http/server.go:1830 +0x651
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2795 +0x27b
10.200.150.121 - - [16/Jul/2019:16:51:10 +0100] "GET /aws/eip/detached HTTP/1.1" 200 1
10.200.150.121 - - [16/Jul/2019:16:51:10 +0100] "GET /aws/ec2/reserved HTTP/1.1" 200 3
2019/07/16 16:51:19 http: panic serving 10.200.150.121:58590: runtime error: invalid memory address or nil pointer dereference
goroutine 12 [running]:
net/http.(*conn).serve.func1(0xc420459b80)
	/usr/local/go/src/net/http/server.go:1726 +0xd0
panic(0x1ea5ee0, 0x448e770)
	/usr/local/go/src/runtime/panic.go:502 +0x229
github.com/mlabouardy/komiser/services/aws.AWS.getClusters(0xc4206ec990, 0xa, 0x25bbbc0, 0xc42099cfc0, 0x25b8a60, 0xc420a50040, 0xc420a00810, 0xc420c50780, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:50 +0xfd
github.com/mlabouardy/komiser/services/aws.AWS.DescribeECS(0xc420a184d0, 0x9, 0x25bbbc0, 0xc42099cfc0, 0x25b8a60, 0xc420a50040, 0xc420a00810, 0xc420c50780, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:20 +0x3b0
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler(0xc42036c520, 0x7f97d931f188, 0xc420a00630, 0xc420d12e00)
	/go/src/github.com/mlabouardy/komiser/handlers/aws/ecs_handler.go:23 +0x2cb
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler-fm(0x7f97d931f188, 0xc420a00630, 0xc420d12e00)
	/go/src/github.com/mlabouardy/komiser/main.go:78 +0x48
net/http.HandlerFunc.ServeHTTP(0xc420337730, 0x7f97d931f188, 0xc420a00630, 0xc420d12e00)
	/usr/local/go/src/net/http/server.go:1947 +0x44
github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc42033b730, 0x7f97d931f188, 0xc420a00630, 0xc420d12e00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux/mux.go:162 +0xed
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.(*cors).ServeHTTP(0xc4204279e0, 0x7f97d931f188, 0xc420a00630, 0xc420fc0200)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/cors.go:51 +0xa2e
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.loggingHandler.ServeHTTP(0x25bb580, 0xc42000e020, 0x25ba920, 0xc4204279e0, 0x25c4a20, 0xc42029e460, 0xc420fc0200)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/handlers.go:69 +0x123
net/http.serverHandler.ServeHTTP(0xc42020ea90, 0x25c4a20, 0xc42029e460, 0xc420fc0200)
	/usr/local/go/src/net/http/server.go:2694 +0xbc
net/http.(*conn).serve(0xc420459b80, 0x25c5420, 0xc42032e840)
	/usr/local/go/src/net/http/server.go:1830 +0x651
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2795 +0x27b
2019/07/16 16:51:20 http: panic serving 10.200.150.121:58601: runtime error: invalid memory address or nil pointer dereference
goroutine 24 [running]:
net/http.(*conn).serve.func1(0xc42009eaa0)
	/usr/local/go/src/net/http/server.go:1726 +0xd0
panic(0x1ea5ee0, 0x448e770)
	/usr/local/go/src/runtime/panic.go:502 +0x229
github.com/mlabouardy/komiser/services/aws.AWS.getClusters(0xc420295660, 0xa, 0x25bbbc0, 0xc4205c9f20, 0x25b8a60, 0xc4207584a0, 0xc420d82db0, 0xc4205bf380, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:50 +0xfd
github.com/mlabouardy/komiser/services/aws.AWS.DescribeECS(0xc420286eb0, 0x9, 0x25bbbc0, 0xc4205c9f20, 0x25b8a60, 0xc4207584a0, 0xc420d82db0, 0xc4205bf380, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:20 +0x3b0
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler(0xc42036c520, 0x7f97d931f188, 0xc420d82ae0, 0xc42038e600)
	/go/src/github.com/mlabouardy/komiser/handlers/aws/ecs_handler.go:23 +0x2cb
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler-fm(0x7f97d931f188, 0xc420d82ae0, 0xc42038e600)
	/go/src/github.com/mlabouardy/komiser/main.go:78 +0x48
net/http.HandlerFunc.ServeHTTP(0xc420337730, 0x7f97d931f188, 0xc420d82ae0, 0xc42038e600)
	/usr/local/go/src/net/http/server.go:1947 +0x44
github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc42033b730, 0x7f97d931f188, 0xc420d82ae0, 0xc42038e600)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux/mux.go:162 +0xed
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.(*cors).ServeHTTP(0xc4204279e0, 0x7f97d931f188, 0xc420d82ae0, 0xc420d13c00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/cors.go:51 +0xa2e
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.loggingHandler.ServeHTTP(0x25bb580, 0xc42000e020, 0x25ba920, 0xc4204279e0, 0x25c4a20, 0xc42056bea0, 0xc420d13c00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/handlers.go:69 +0x123
net/http.serverHandler.ServeHTTP(0xc42020ea90, 0x25c4a20, 0xc42056bea0, 0xc420d13c00)
	/usr/local/go/src/net/http/server.go:2694 +0xbc
net/http.(*conn).serve(0xc42009eaa0, 0x25c5420, 0xc42007a200)
	/usr/local/go/src/net/http/server.go:1830 +0x651
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2795 +0x27b
2019/07/16 16:51:21 http: panic serving 10.200.150.121:58597: runtime error: invalid memory address or nil pointer dereference
goroutine 19 [running]:
net/http.(*conn).serve.func1(0xc42009e6e0)
	/usr/local/go/src/net/http/server.go:1726 +0xd0
panic(0x1ea5ee0, 0x448e770)
	/usr/local/go/src/runtime/panic.go:502 +0x229
github.com/mlabouardy/komiser/services/aws.AWS.getClusters(0xc4205182e0, 0xa, 0x25bbbc0, 0xc420900ae0, 0x25b8a60, 0xc4202db760, 0xc42071a870, 0xc420031900, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:50 +0xfd
github.com/mlabouardy/komiser/services/aws.AWS.DescribeECS(0xc4205772e0, 0x9, 0x25bbbc0, 0xc420900ae0, 0x25b8a60, 0xc4202db760, 0xc42071a870, 0xc420031900, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:20 +0x3b0
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler(0xc42036c520, 0x7f97d931f188, 0xc42071a630, 0xc42038fb00)
	/go/src/github.com/mlabouardy/komiser/handlers/aws/ecs_handler.go:23 +0x2cb
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler-fm(0x7f97d931f188, 0xc42071a630, 0xc42038fb00)
	/go/src/github.com/mlabouardy/komiser/main.go:78 +0x48
net/http.HandlerFunc.ServeHTTP(0xc420337730, 0x7f97d931f188, 0xc42071a630, 0xc42038fb00)
	/usr/local/go/src/net/http/server.go:1947 +0x44
github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc42033b730, 0x7f97d931f188, 0xc42071a630, 0xc42038fb00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux/mux.go:162 +0xed
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.(*cors).ServeHTTP(0xc4204279e0, 0x7f97d931f188, 0xc42071a630, 0xc420d13b00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/cors.go:51 +0xa2e
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.loggingHandler.ServeHTTP(0x25bb580, 0xc42000e020, 0x25ba920, 0xc4204279e0, 0x25c4a20, 0xc420237180, 0xc420d13b00)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/handlers.go:69 +0x123
net/http.serverHandler.ServeHTTP(0xc42020ea90, 0x25c4a20, 0xc420237180, 0xc420d13b00)
	/usr/local/go/src/net/http/server.go:2694 +0xbc
net/http.(*conn).serve(0xc42009e6e0, 0x25c5420, 0xc42007a3c0)
	/usr/local/go/src/net/http/server.go:1830 +0x651
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2795 +0x27b
2019/07/16 16:51:21 http: panic serving 10.200.150.121:58608: runtime error: invalid memory address or nil pointer dereference
goroutine 1138 [running]:
net/http.(*conn).serve.func1(0xc420eac000)
	/usr/local/go/src/net/http/server.go:1726 +0xd0
panic(0x1ea5ee0, 0x448e770)
	/usr/local/go/src/runtime/panic.go:502 +0x229
github.com/mlabouardy/komiser/services/aws.AWS.getClusters(0xc420039ac0, 0xa, 0x25bbbc0, 0xc420f3ccc0, 0x25b8a60, 0xc420307da0, 0xc420f2e930, 0xc420283a00, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:50 +0xfd
github.com/mlabouardy/komiser/services/aws.AWS.DescribeECS(0xc42039b200, 0x9, 0x25bbbc0, 0xc420f3ccc0, 0x25b8a60, 0xc420307da0, 0xc420f2e930, 0xc420283a00, 0x2, 0x5, ...)
	/go/src/github.com/mlabouardy/komiser/services/aws/ecs.go:20 +0x3b0
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler(0xc42036c520, 0x7f97d931f188, 0xc420f2e720, 0xc420d80400)
	/go/src/github.com/mlabouardy/komiser/handlers/aws/ecs_handler.go:23 +0x2cb
github.com/mlabouardy/komiser/handlers/aws.(*AWSHandler).ECSHandler-fm(0x7f97d931f188, 0xc420f2e720, 0xc420d80400)
	/go/src/github.com/mlabouardy/komiser/main.go:78 +0x48
net/http.HandlerFunc.ServeHTTP(0xc420337730, 0x7f97d931f188, 0xc420f2e720, 0xc420d80400)
	/usr/local/go/src/net/http/server.go:1947 +0x44
github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc42033b730, 0x7f97d931f188, 0xc420f2e720, 0xc420d80400)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/mux/mux.go:162 +0xed
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.(*cors).ServeHTTP(0xc4204279e0, 0x7f97d931f188, 0xc420f2e720, 0xc420d80200)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/cors.go:51 +0xa2e
github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers.loggingHandler.ServeHTTP(0x25bb580, 0xc42000e020, 0x25ba920, 0xc4204279e0, 0x25c4a20, 0xc4201dfc00, 0xc420d80200)
	/go/src/github.com/mlabouardy/komiser/vendor/github.com/gorilla/handlers/handlers.go:69 +0x123
net/http.serverHandler.ServeHTTP(0xc42020ea90, 0x25c4a20, 0xc4201dfc00, 0xc420d80200)
	/usr/local/go/src/net/http/server.go:2694 +0xbc
net/http.(*conn).serve(0xc420eac000, 0x25c5420, 0xc420f63780)
	/usr/local/go/src/net/http/server.go:1830 +0x651
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2795 +0x27b
```

**Comment by CyrilAll** [comment]
getClusters might be the issue (verify that the permission is not missing). There might be a lack of permission, and the error is not yet well managed.
Could you check if you have added the permission getClusters to the iam role? Thanks
@guidorugo

**Comment by mlabouardy** [comment]
we've made a fix with the latest release (v2.5.0)
